lib/rules/eol-last.js

```diff
@@ -1,62 +1,95 @@
 /**
- * @fileoverview Require file to end with single newline.
+ * @fileoverview Require or disallow file to end with single newline.
  * @author Nodeca Team <https://github.com/nodeca>
+ * @author kdex <https://github.com/kdex>
```

Comment:
@kdex Yes, probably. Maybe we'll sweep through those later. (Some cases are legitimate collaborations at time of file creation, though.) Thanks for your understanding!